projects/addon-charts/package.json

```diff
@@ -16,7 +16,7 @@
     "peerDependencies": {
         "@angular/common": ">=9.0.0",
         "@angular/core": ">=9.0.0",
-        "@tinkoff/ng-polymorpheus": "3.1.9",
+        "@tinkoff/ng-polymorpheus": "3.1.10",
```

Comment:
please, bump to 3.1.11 for every package for align peerDependencies

Reply:
Done